src/components/sbb-toggle-check/sbb-toggle-check.tsx

```diff
@@ -109,45 +109,53 @@ export class SbbToggleCheck implements ComponentInterface {
       .some((n: Node) => !!n.textContent.trim());
   }
 
-  public render(): JSX.Element {
+  protected override render(): TemplateResult {
     const attributes = {
       role: 'checkbox',
       'aria-checked': this.checked?.toString() ?? 'false',
       'aria-required': this.required.toString(),
       'aria-disabled': this.disabled.toString(),
       ...(this.disabled ? undefined : { tabIndex: '0' }),
     };
-    return (
-      <Host {...attributes}>
-        <label class="sbb-toggle-check">
-          <input
-            type="checkbox"
-            aria-hidden="true"
-            tabIndex={-1}
-            name={this.name}
-            disabled={this.disabled}
-            required={this.required}
-            checked={this.checked}
-            value={this.value}
-            onInput={() => this.handleInputEvent()}
-            onChange={(event) => this.handleChangeEvent(event)}
-          />
-          <span class="sbb-toggle-check__container">
-            <span class="sbb-toggle-check__label" hidden={!this._hasLabelText}>
-              <slot onSlotchange={(event): void => this._onLabelSlotChange(event)} />
-            </span>
-            <span class="sbb-toggle-check__track">
-              <span class="sbb-toggle-check__circle">
-                <span class="sbb-toggle-check__icon">
-                  <slot name="icon">
-                    {this.iconName && <sbb-icon name={this.iconName}></sbb-icon>}
-                  </slot>
-                </span>
+
+    setAttributes(this, attributes);
+
+    return html`
+      <label class="sbb-toggle-check">
+        <input
+          type="checkbox"
+          aria-hidden="true"
+          tabindex="-1"
+          .name=${this.name || nothing}
```

Comment:
The `|| nothing` should be redundant if you're setting the property

Reply:
If I remove that the spec test fails because of an extra `name=""`, so I guess we can keep it

Reply:
Ok, good to know. Thanks ❤️